riscv/build.rs

```diff
@@ -12,6 +12,7 @@ fn main() {
 }
 
 fn build_lalrpop() {
+    println!("cargo:rerun-if-changed=src/riscv_asm.lalrpop");
```

Comment:
Do we really need that?

Reply:
Yes. It is like the other `println!` in this file. Without this, `cargo build` will not re-run `build.rs` if lalrpop file is changed. It took me a few runs to notice the lalrpop file was not being recompiled.

Reply:
This is an interior feature of lalrpop. We should not need to care about that. Documentation is here: https://lalrpop.github.io/lalrpop/advanced_setup.html#rerun-directives

Reply:
I was not trying to prevent it from being executed every time, I was trying to get it to run in the first place.

Once there is at least one `cargo:rerun-if` directive output in the `build.rs`, it will only run if any of the condition matches. The `build_instruction_tests()` has, so now `build_lalrpop()` has to have it, too.

Reply:
But does that mean we also need it for the general powdr grammar?

Reply:
Also is there no way to replace this by a function call on `lalrpop`? I'm just wondering because now we need to change this line if we ever change the grammar file name.

Reply:
> But does that mean we also need it for the general powdr grammar?

No, because `parser/src/build.rs` is always executed, because there is no `cargo:rerun-if-...` directive being output there.

> Also is there no way to replace this by a function call on lalrpop?

Yes, let me change that.

Reply:
Done.